crates/hir_def/src/nameres.rs

```diff
@@ -258,6 +258,10 @@ impl DefMap {
         self.krate
     }
 
+    pub(crate) fn block_id(&self) -> Option<BlockId> {
+        self.block.as_ref().map(|block| block.block)
```

Comment:
Nit: `Some(self.block?.block)`

But I don't think it matters here and probably slows down the compiler.